widgetry/src/style/mod.rs

```diff
@@ -77,19 +78,22 @@ impl Style {
         Style {
             outline_thickness: 2.0,
             outline_color: Color::WHITE,
-            panel_bg: Color::grey(0.4),
             hotkey_color: Color::GREEN,
             loading_tips: Text::new(),
 
             // Buttons
-            btn_solid_floating: ButtonStyle::btn_solid_floating(),
+            btn_solid: ButtonStyle::btn_solid(),
 
             // legacy day theme
+            panel_bg: Color::grey(0.4),
+            text_fg_color: Color::WHITE,
             btn_outline: ButtonStyle::btn_outline(),
-            btn_solid: ButtonStyle::btn_solid(),
+            btn_solid_floating: ButtonStyle::btn_solid_floating(),
             // TODO new day theme
```

Comment:
This PR is another "no-op" for the user (hopefully)
<img width="1072" alt="Screen Shot 2021-02-23 at 5 43 19 PM" src="https://user-images.githubusercontent.com/217057/108932987-2868d000-75ff-11eb-8603-3bc121c7aa54.png">

But you can see the day theme starting to take shape if you toggle this commented out section.
<img width="2032" alt="Screen Shot 2021-02-23 at 5 49 15 PM" src="https://user-images.githubusercontent.com/217057/108933124-754ca680-75ff-11eb-874f-58e604cb2d47.png">

Reply:
It's starting to come together!